freeipa/src/test/java/com/sequenceiq/freeipa/service/freeipa/user/UserSyncServiceTest.java

```diff
@@ -159,6 +188,34 @@ void testRemoveUsersFromGroupsPartitionsRequests() throws Exception {
         });
     }
 
+    @Test
+    void testAsyncSynchronizeUsersUsesInternalCrn() {
+        Stack stack = mock(Stack.class);
+        when(stack.getEnvironmentCrn()).thenReturn(ENV_CRN);
+        when(stackService.getMultipleByEnvironmentCrnOrChildEnvironmantCrnAndAccountId(anySet(), anyString())).thenReturn(List.of(stack));
```

Comment:
'stackService.getMultipleByEnvironmentCrnOrChildEnvironmantCrnAndAccountId' was renamed in rc-2.19. This test is new, so it merged fine.